### PR TITLE
Add pulsebot to #gfx

### PR DESCRIPTION
**Pull request by staktrace**
I'd like pulsebot to report on commits to hg.m.o/projects/graphics in #gfx (and update bugs with the commits as well).

### CODE REVIEW

File: pulsebot.cfg.in [hunk 1]
```diff
@@ -32,5 +33,5 @@ server = https://api.pub.build.mozilla.org/treestatus/trees
 server = https://bugzilla.mozilla.org/
 user = @bugzilla.user@
 password = @bugzilla.password@
-pulse = integration/b2g-inbound,integration/fx-team,integration/mozilla-inbound,integration/autoland,mozilla-central,hgcustom/version-control-tools,mozilla-build,webtools/reviewboard
+pulse = integration/b2g-inbound,integration/fx-team,integration/mozilla-inbound,integration/autoland,mozilla-central,hgcustom/version-control-tools,mozilla-build,webtools/reviewboard,projects/graphics
```
Comment (glandium) [inline pulsebot.cfg.in]: Do you want bugs landed on that project branch to be closed?

Comment (glandium) [inline pulsebot.cfg.in:36]: Do you want bugs landed on that project branch to be closed?

Comment (staktrace) [reply pulsebot.cfg.in:36]: Yup

Comment (staktrace) [reply pulsebot.cfg.in:36]: (For now we're not merging to m-c, so it makes sense to close them when they land on graphics)

**Pull request merged**